src/components/Page-Title/Page-Title.js

```diff
@@ -27,7 +27,7 @@ export default class PageTitle extends React.Component {
           </span>
         </div>
       </a>
-    ) : null;
+    ) : <a className="backlink-padding"></a>;
```

Comment:
I understand this is necessary to reserve the blank space when the backlink is not provided. Can you add a custom class with fixed height, something like - `backlink-slot`or `backlink-blank`, instead, please?

Reply:
<a className="chi-link -backlink-blank"></a>;